core/modules/node/node.module

```diff
@@ -75,6 +75,10 @@ const NODE_ACCESS_IGNORE = NULL;
  * Implements hook_theme().
  */
 function node_theme() {
+  // Include the theme file so that the base process/preprocess functions are
+  // in the global namespace.
+  module_load_include('inc', 'node', 'node.theme');
+
```

Comment:
This shouldn't be necessary. The file should be loaded as long as each entry in hook_theme() specifies the `node.theme.inc` file. You should include `file => 'node.theme.inc'` even for template files, the 'file' will be loaded for preprocess functions, and the template will be used for the actual rendering.